src/marines/automation/create_project.py

```diff
@@ -0,0 +1,61 @@
+import json
+import requests
+import pprint
+
+#class AutomationForTodoLy():
+
+host = "https://todo.ly/api/"
+#def create_project():
+endpoint = host + "projects.json"
+header = {"Authorization": "Basic Z2F0aXRhMDEwMS5teG1AZ21haWwuY29tOjEyMzQ1Njc4",
```

Comment:
do not use credentials or do not push credentials

Reply:
this is only class example

Reply:
fixed